action/rlp_tx.go

```diff
@@ -121,23 +115,27 @@ func DecodeEtherTx(rawData string) (*types.Transaction, error) {
 func ExtractTypeSigPubkey(tx *types.Transaction) (iotextypes.Encoding, []byte, crypto.PublicKey, error) {
 	var (
 		encoding iotextypes.Encoding
-		signer   types.Signer
+		signer   = types.NewEIP2930Signer(tx.ChainId()) // by default assume latest signer
 		V, R, S  = tx.RawSignatureValues()
 	)
 	// extract correct V value
 	switch tx.Type() {
 	case types.LegacyTxType:
 		if tx.Protected() {
 			chainIDMul := tx.ChainId()
-			V = new(big.Int).Sub(V, chainIDMul.Lsh(chainIDMul, 1))
+			V = new(big.Int).Sub(V, new(big.Int).Lsh(chainIDMul, 1))
```

Comment:
`iotextypes.Encoding_ETHEREUM_ACCESSLIST` could be optional since len(tx.accesslist) > 0 is used as the type selector